tests/forms_tests/field_tests/test_durationfield.py

```diff
@@ -20,6 +20,15 @@ def test_durationfield_clean(self):
             datetime.timedelta(days=1, hours=1, minutes=15, seconds=30, milliseconds=300),
             f.clean('1 1:15:30.3')
         )
+        self.assertEqual(datetime.timedelta(0, 10800), f.clean(datetime.timedelta(0, 10800)))
+        msg = 'This field is required.'
+        with self.assertRaisesMessage(ValidationError, msg):
```

Comment:
I think we don't need to define `msg` variable:
```suggestion
        with self.assertRaisesMessage(ValidationError, 'This field is required.'):
```